doc/telemetry.md

```diff
@@ -0,0 +1,102 @@
+# Telemetry
+
+The SPIRE Server and Agent can be configured to emit metrics that can be send to the supported metrics collectors. For instructions on how to configure them properly please refer to the [Telemetry Configuration](telemetry_config) guide.
```

Comment:
The .md extension is missing in the link.

```suggestion
The SPIRE Server and Agent can be configured to emit metrics that can be sent to the supported metrics collectors. For instructions on how to configure them properly, please refer to the [Telemetry Configuration](telemetry_config.md) guide.
```

Reply:
good catch!